scripts/sayhi.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports = (robot) ->
+   robot.respond /say \"([^\"]*)\" to (.*)$/i, (msg) ->
+     msg.send "" + msg.match[2] + ": " msg.match[1]
```

Comment:
Hey @teroxik, you didn't reviewed the code carefully before merging PR :-1:  , miss the symbol "+" at this line:
```
": " msg.match[1]
```